sdk/eventhub/test-resources.bicep

```diff
@@ -32,6 +38,7 @@ resource eventHubNamespace 'Microsoft.EventHub/namespaces@2024-01-01' = {
     zoneRedundant: false
     isAutoInflateEnabled: false
     maximumThroughputUnits: 0
+    disableLocalAuth: subscription().id != TMESubId
```

Comment:
Do the tests work when this is disabled?

Reply:
when `disableLocalAuth: true`, I get the following error:

> an error occurred while establishing the links: MessagingError: LocalAuthDisabled: Authorization failed because SAS authentication has been disabled for the namespace